Model/Config/Source/Status/ProcessingMaintain.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Adyen\Payment\Model\Config\Source\Status;
+
+use Adyen\Payment\Helper\Webhook;
+
+/**
+ * Order Statuses source model
+ */
+class ProcessingMaintain extends \Magento\Sales\Model\Config\Source\Order\Status
+{
+    /**
+     * @var string[]
+     */
+    protected $_stateStatuses = [
```

Comment:
Could you set these data members to private and remove the underscore?

Reply:
@tnaber this one still needs resolving